code/modules/mob/new_player/new_player.dm

```diff
@@ -107,8 +107,7 @@
 
 			if(alert(src,"Are you sure you wish to observe? You will have to wait 30 minutes before being able to respawn!","Player Setup","Yes","No") == "Yes")
 				if(!client)	return 1
-				var/mob/dead/observer/observer = new()
-
+				var/mob/dead/observer/observer = new /mob/dead/observer(src)

```

Comment:
For future ref, `var/mob/dead/observer/observer = new(src)` gives the same result. No need to change atm, tho.